app/views/govuk_publishing_components/components/_layout_footer.html.erb

```diff
@@ -89,6 +90,19 @@
           data-track-label="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
           data-track-options='{"dimension29": "Open Government Licence v3.0"}'
           data-track-links-only
+          data-ga4-track-links-only
+          data-ga4-link="<%= {
+            "event_name": "navigation",
+            "section": "Licence",
+            "index": {
+              "index_section": "4",
+              "index_link": "1",
+              "index_section_count": "5",
```

Comment:
Nitpick: do these numbers need to be quoted?

Reply:
@andysellick I think in our docs we had the data type of all our numbers as a string. Our code does work without them being a string but I thought I would be consistent with our documentation even though we do use a regular number in a lot of places. Is this OK?